esrally/utils/io.py

```diff
@@ -405,16 +395,20 @@ def _do_decompress_manually_with_lib(target_directory: str, filename: str, compr
         compressed_file.close()
 
 
-def _do_decompress(target_directory: str, compressed_file: Union[zipfile.ZipFile, tarfile.TarFile]) -> None:
+def _do_tar_decompress(target_directory: str, compressed_file: tarfile.TarFile) -> None:
+    try:
+        compressed_file.extractall(path=target_directory, filter="tar")
```

Comment:
it is now recommended to pass `filter` kwarg when calling `extractall()` for tar files